source/src/components/html_fragments/README.md

```diff
@@ -0,0 +1,6 @@
+# HTML Fragments for the components
+
+Currently unused except for tests rather than rewriting, document initialization
```

Comment:
Not exactly. These files are used in the unit tests. We'd need a more detailed build process in order to include them dynamically into the website. I don't know of any good static site generators that doesn't require us to add some Ruby gem and add another thing into our toolchain. Ideally it would be a static site generator so that it doesn't need to be hosted and for performance sake.